### ISSUE TITLE
synology 7.2 Not supported, display access denied

### ISSUE THREAD

**Issue by baifengs** [erickok/transdroid#668]
synology 7.2 Not supported, display access denied。
from china

群晖7系统，不支持，显示访问被拒绝。

**Comment by erickok** [comment]
Indeed, 7.2 is not supported. I also don't have access to such a device so it will be hard to support it. I'll keep this open because ideally we would support it indeed.
